src/attr/exceptions.pyi

```diff
@@ -1,7 +1,14 @@
+from typing import Any
+
 class FrozenInstanceError(AttributeError):
     msg: str = ...
 
 class AttrsAttributeNotFoundError(ValueError): ...
 class NotAnAttrsClassError(ValueError): ...
 class DefaultAlreadySetError(RuntimeError): ...
 class UnannotatedAttributeError(RuntimeError): ...
+
+class NotCallableError(TypeError):
+    message: str = ...
```

Comment:
Since `FrozenInstanceError` already is using `msg`, we should stay consistent here.